src/main/java/nextstep/subway/favorite/ui/FavoriteController.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.subway.favorite.ui;
+
+import nextstep.subway.auth.domain.AuthenticationPrincipal;
+import nextstep.subway.auth.domain.LoginMember;
+import nextstep.subway.favorite.application.FavoriteService;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import nextstep.subway.member.dto.MemberRequest;
+import nextstep.subway.member.dto.MemberResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class FavoriteController {
+    private FavoriteService favoriteService;
+
+    @PostMapping("/favorites")
+    public ResponseEntity createFavorite(@AuthenticationPrincipal LoginMember loginMember, @RequestBody FavoriteRequest request) {
```

Comment:
```suggestion
    public ResponseEntity<Void> createFavorite(@AuthenticationPrincipal LoginMember loginMember, @RequestBody FavoriteRequest request) {
```

return body 가 없을때도 이런식으로 명시하는게 좋은 것 같더라구요 :)